### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by GoranGozo** [javaguirre/comment#6]
[Enter steps to reproduce below:]

1. Open Apache 2 httpd-vhosts.conf
2. In menu click "Packages" -> "Comments" -> "Toggle"

**Atom Version**: 0.165.0
**System**: Mac OS X 10.9.5
**Thrown From**: [comment](https://github.com/havber/comment) package, v0.2.7

### Stack Trace

Uncaught TypeError: Cannot read property 'length' of undefined

```
At /Users/goran/.atom/packages/comment/lib/comment.js:170

TypeError: Cannot read property 'length' of undefined
  at supportsMultiline (/Users/goran/.atom/packages/comment/lib/comment.js:170:41)
  at atom-workspace.module.exports.toggle (/Users/goran/.atom/packages/comment/lib/comment.js:31:17)
  at atom-workspace.handler (/Applications/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:156:19)
  at Ipc.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:65:30)
  at Ipc.emit (events.js:107:17)
  at process.<anonymous> (/Applications/Atom.app/Contents/Resources/atom/renderer/api/lib/ipc.js:22:29)
  at process.emit (events.js:110:17)

```

### Commands

```
     -2:54.3 find-and-replace:show (input.hidden-input)
     -2:54.0 core:paste (input.hidden-input)
  4x -2:53.4 core:confirm (input.hidden-input)
     -2:23.9 core:move-down (input.hidden-input)
     -2:23.7 core:move-left (input.hidden-input)
 26x -2:23.3 core:select-down (input.hidden-input)
  5x -2:20.6 core:select-up (input.hidden-input)
     -2:18.4 core:select-down (input.hidden-input)
     -2:10.0 grammar-selector:show (atom-text-editor.editor.editor-colors.null.grammar.is-focused)
     -1:03.3 command-palette:toggle (input.hidden-input)
  2x -0:59.4 core:move-down (input.hidden-input)
     -0:59.0 core:confirm (input.hidden-input)
     -0:59.0 settings-view:install-packages (input.hidden-input)
  2x -0:53.3 core:confirm (input.hidden-input)
     -0:00.0 comment:toggle (input.hidden-input)
```

### Config

```
{}
```

### Installed Packages

```coffee
# User
emmet, v2.3.0
jsformat, v0.7.18
jshint, v1.1.3
jslint, v0.10.2
pretty-json, v0.3.1
remote-sync, v2.3.1
trailing-spaces, v0.2.3
view-tail-large-files, v0.1.6

# Dev
No dev packages
```

/cc @atom/core

**Comment by scelera** [comment]
This seems to happen when the file hasn't been save with a type (html, js) or if the file type isn't supported by this package (py).

**Comment by VeeeneX** [comment]
Yes, same proble with php :smile:

**Comment by javaguirre** [comment]
Hi! This should be fixed on a5393529b0dbd676b1e85b1d40dbc0b922452911

Please let me know if It persists!

Thank you!
